civihr_employee_portal/tests/Helpers/LoggingStreamSwitcherTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Drupal\civihr_employee_portal\Helpers\LoggingStreamSwitcher;
+
+class LoggingStreamSwitcherTest extends \PHPUnit_Framework_TestCase {
+
+  public function testSwitchingOnWillChangeGlobalVariable() {
+    LoggingStreamSwitcher::enableLogging('foo');
+    global $civihrChangeLogStreams;
+
+    $this->assertTrue($civihrChangeLogStreams['foo']);
+  }
+
+  public function testDisablingWillSetGlobalVariableToFalse() {
+    LoggingStreamSwitcher::disableLogging('foo');
+    global $civihrChangeLogStreams;
+
+    $this->assertFalse($civihrChangeLogStreams['foo']);
+  }
+
+  public function testSwitchingWillChangeGlobalVariable() {
+    LoggingStreamSwitcher::enableLogging('foo');
```

Comment:
Since we are testing the switching, would be good to have an assert here: `$this->assertTrue(LoggingStreamSwitcher::isEnabled('foo'))` to check if it is true in addition to checking if it is false after the switch.